src/generate-report.ts

```diff
@@ -184,7 +185,7 @@ class GenerateCtrfReport implements Reporter {
       }
 
       if (testResult.failureDetails !== undefined) {
-        failureDetails.trace = testResult.failureMessages.join('\r\n')
+        failureDetails.trace = failureDetails.trace?.concat('\n\n', testResult.failureDetails.join('\n'))
```

Comment:
Originally, `failureMessage` was assigned to trace. Was this intended?
And is it possible to just join `failureDetails` which is `Array<unknown>`. I don't know how to handle this properly